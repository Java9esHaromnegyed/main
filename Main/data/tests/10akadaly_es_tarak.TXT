loadArena testArena10.txt
initArena
robotMovement 1 OIL
tick 1
robotMovement 1 PUTTY
tick 1
robotMovement 1 OIL
tick 1
robotMovement 1 PUTTY
tick 1
robotMovement 1 OIL
tick 1
robotMovement 1 PUTTY