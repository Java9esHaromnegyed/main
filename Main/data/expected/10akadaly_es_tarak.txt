arenaLoaded testArena10.txt
robotAdded id: 1 name: „player1” pos: [0; 0]
arenaInited
obstacleAdded type: OIL pos: [0; 0]
tick
robotMoved id: 1 pos: [1; 0]
obstacleAdded type: PUTTY pos: [1; 0]
tick
robotMoved id: 1 pos: [2; 0]
#OIL TANK IS EMPTY
#PUTTY TANK IS EMPTY